libraries/aleth/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+### Aleth part
```

Comment:
hmm when we finally have these sub-libraries. Do we even need aleth interface ? We could link separate devcore devcrypto p2p libs when needed, so it should be even more effective...

Reply:
I will check it one more time, but I think we are using all this libs in common. So all other libs also need them all

Reply:
removed it